extensions/notebook/resources/notebooks/JupyterBooksCreate.ipynb

```diff
@@ -71,24 +71,27 @@
             "source": [
                 "import os, re, shutil\r\n",
                 "\r\n",
-                "overwrite = False\r\n",
-                "book_name = input('Please provide the path where the book needs to be saved along with the book name ex-> D:\\Book1: ') \r\n",
+                "try:\r\n",
+                "    overwrite = False\r\n",
+                "    book_name = input('Please provide the path where the book needs to be saved along with the book name ex-> D:\\Book1: ') \r\n",
```

Comment:
This would be a lot better if we could use a file picker dialog - have we done anything like that before from a Notebook?

Reply:
Will look into that. Thanks.